css/buttons.styl

```diff
@@ -52,7 +52,7 @@
   padding: 0 0.8em 0 0.5em
   vertical-align: middle
 
-.on-home-page, .on-home-page-not-logged-in, .on-landing-page
+.on-home-page, .on-home-page-not-logged-in, .on-landing-page, .on-home-page-logged-in
```

Comment:
Can we just use the `on-home-page` class and just make sure the home page root uses that class?